.github/workflows/_tests.yml

```diff
@@ -16,6 +16,7 @@ on:
           "packages/paywall",
           "smart-contracts",
           "smart-contract-extensions",
+          "subgraph",
```

Comment:
Nice!

Reply:
the CI only runs on master though so it may fail when we merged :/

Reply:
ha, want to create a first PR that has just that so we can merge it and make sure it runs on this PR?

Reply:
let's keep it and revert if this breaks master!